project/project.go

```diff
@@ -65,15 +65,18 @@ func NewProject(context *Context, runtime RuntimeProject, parseOptions *config.P
 	}
 
 	if context.EnvironmentLookup == nil {
-		cwd, err := os.Getwd()
+		absPath, err := filepath.Abs(context.ComposeFiles[0])
```

Comment:
The `app` package guarantees that `len(context.ComposeFiles) > 0`, but it's possible to create a project using `ComposeBytes` rather than `ComposeFiles`.